adoc/deployment-preparation.adoc

```diff
@@ -4,37 +4,59 @@
+load it into your users `ssh-agent` program. This is also mandatory in order to be able to use
+the installation tools `terraform` and `skuba`. In a later deployment step,
+skuba will ensure that the key is distributed across all the nodes and trusted by them.
+For now, you only need to make sure that an ssh-agent is running and that it has
+the SSH key added:
+
+. The `ssh-agent` is usually started automatically by graphical
+desktop environments. If that is not your case run:
++
+----
+eval "$(ssh-agent)"
+----
+This will start the agent and set environment variables used for agent
+communication within the current session. This has to be the same terminal session
+that you run the `skuba` commands in. A new terminal usually requires a new ssh-agent.
+. In some desktop environments the ssh-agent will also automatically load the SSH keys.
+To add an SSH keys manually, use the `ssh-add` command:
++
+----
+ssh-add <path_to_key>
+----
++
 [TIP]
 ====
-The `ssh-agent` process is usually started automatically by most UNIX
-environments. If that's not the case, invoke the `ssh-agent` command
-and follow the guidance given by the tool's output.
-
-You can start the `ssh-agent` in the background by using the
-`eval "$(ssh-agent -s)"` command.
-
-You can load as many keys as you want into your agent using the
-`ssh-add <path to key>` command. Keys can also be password protected.
+If you are adding the SHH key manually, specify the full path.
+For example: `/home/sles/.ssh/id_rsa`
+====
 
-You can list all the identities loaded into the agent by using the
-`ssh-add -L` command.
+You can load multiple keys into your agent using the `ssh-add <path_to_key>` command.
+Keys should be password protected for security measures. The
+ssh-add command will prompt for your password, then the agent caches the
+private key for a configurable lifetime. The `-t lifetime` option to
+ssh-add specifies a maximum time to cache the key. See `man ssh-add` for
+more information.
 
-skuba will try all the identities loaded into the ssh-agent until one of
-them grants access to the node.
+[NOTE]
+====
+Skuba will try all the identities loaded into the ssh-agent until one of
+them grants access to the node, or until the ssh server maximum
+authentication attempts are exhausted.
+====
 
-It is also possible to forward the authentication agent connection from a
-host to another one. This can be useful if you intend to run skuba on
+==== Forwarding the authentication agent connection
+It is also possible to *forward the authentication agent connection* from a
+host to another one, which can be useful if you intend to run skuba on
 a "jump host" and don't want to copy your private key to this node.
-
 This can be achieved using the `ssh -A` command. Please refer to the man page
-of ssh to learn about the security implications of using this feature.
-====
+of SSH to learn about the security implications of using this feature.
```

Comment:
This is referring to the command `ssh`, not the protocol "SSH", so lower-case might be more accurate.  Maybe?  I'm somewhat indifferent.

Reply:
usually man pages are referred to in lowercase, or even more specifically `ssh(1)`.